### ISSUE TITLE
Set up an automated ML task to predict drug -> drug target edges

### ISSUE THREAD

**Issue by justaddcoffee** [Knowledge-Graph-Hub/kg-idg#43]
### Describe the desired behavior

It would be useful for Drug Central to have inferred drug -> drug target edges, which we could formulate as a graph ML link prediction task. 

We maybe could/should do this in an automated way, on each build of KG-IDG. 

A possible roadmap: 

- Write Jupyter notebook for link prediction task
- Write some tooling to run this code from Jenkins
- Write some code to push the results to KGHub

## Additional context

Per convo with Tudor et al on IDG just now

**Comment by justaddcoffee** [comment]
Discussing this more with Harry - some thoughts. 

We can use NEAT for this (i.e. spin up a Google Cloud instance from Jenkins, pass off a YAML file to the instance describing the graph ML task, wait for it to complete and upload artifacts to S3). 

This strategy has the advantage of: 

- helping us get NEAT updated
- helping us flesh out the general use case of getting automatic embedding and graph ML working with KGHub and other projects (Monarch, eco-kg, others)

**Comment by caufieldjh** [comment]
As per IDG meeting on Jan 20, following drug categories would be good for evaluation:
* Those targeting GPCRs
* antihistaminics (H1 blockers)
* antipsychotics (multi-target)
* antivirals (no human targets)
* antiestrogens (mostly ERalpha)

**Comment by caufieldjh** [comment]
I don't think we include any viral proteins right now, so we'd expect the antivirals to have few/no human targets, especially because we aren't doing any inference based off sequence or structure.
